imports/plugins/core/shipping/client/containers/parcelSizeSettings.js

```diff
@@ -0,0 +1,117 @@
+import { Meteor } from "meteor/meteor";
+import { registerComponent, composeWithTracker } from "@reactioncommerce/reaction-components";
+import { Reaction, i18next, Logger } from "/client/api/";
+import { Shops } from "/lib/collections";
+import { ShippingParcel } from "/lib/collections/schemas";
+import { convertWeight, convertLength } from "/lib/api";
+import ParcelSizeSettings from "../components/parcelSizeSettings";
+
+/**
+ * @method saveDefaultSize
+ * @summary call "shipping/updateParcelSize" method
+ * @param {Object} size - size object to be saved
+ * @since 1.1.12
+ * @return {Function} callback
+*/
+const saveDefaultSize = (size) => {
+  const parcel = ShippingParcel.clean(size);
+  Object.keys(parcel).forEach((key) => {
+    parcel[key] = Number(parcel[key]);
+  });
+  try {
+    ShippingParcel.validate(parcel);
+  } catch (error) {
+    Logger.error(error);
+    Alerts.toast(i18next.t("shippingSettings.parcelSize.saveFailed"), "error");
+    return;
+  }
+  Meteor.call("shipping/updateParcelSize", parcel, (error) => {
+    if (error) {
+      Alerts.toast(i18next.t("shippingSettings.parcelSize.saveFailed"), "error");
+    }
+  });
+};
+
+/**
+ * @method onCardExpand
+ * @summary set "edit/focus" in current Reaction state
+ * @param {String} cardName - card name to be set
+ * @since 1.1.12
+ * @return {Function} callback
+*/
+const onCardExpand = (cardName) => {
+  Reaction.state.set("edit/focus", cardName);
+};
+
+/**
+ * @method getEditFocus
+ * @summary get "edit/focus" value from current Reaction state
+ * @since 1.1.12
+*/
+const getEditFocus = () => Reaction.state.get("edit/focus");
+
+/**
+ * @method composer
+ * @summary composer - reactive Tracker wrapped function
+ * @param {Object} props
+ * @param {Function} onData
+ * @since 1.1.12
+*/
+const composer = (props, onData) => {
+  const shop = Shops.findOne({
+    _id: Reaction.getShopId()
+  });
+  const { baseUOM, baseUOL } = shop;
+  let doc = shop.defaultParcelSize;
+  if (!doc) {
+    doc = {
+      weight: convertWeight("lb", baseUOM, 8),
+      height: convertLength("in", baseUOL, 6),
+      length: convertLength("in", baseUOL, 11.25),
+      width: convertLength("in", baseUOL, 8.75)
+    };
+    // Save default settings for the first time user.
+    saveDefaultSize(doc);
+  }
```

Comment:
If we need this, I'd rather see it in server code. But why do we need it? You have the migration and fixtures updated. `shop/createShop` method actually just clones an existing one. If they import a shop without it, that's not our problem.